### PR TITLE
Use glide instead of godep

### PR DESCRIPTION
**Pull request by zoumo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ before_install:
   # - chmod +x docker-compose
   # - sudo mv docker-compose /usr/local/bin
   # install golang deps.
-  - go get github.com/tools/godep
+  # - go get github.com/tools/godep
```
Comment (superxi911) [inline .travis.yml]: delete

File: caicloud_e2e-test.yml [hunk 2]
```diff
@@ -36,7 +35,7 @@ integration:
     - REGISTRY_LOCATION=cargo.caicloud.io
     - REGISTRY_USERNAME=cyclone
     - REGISTRY_PASSWORD=Cyclone123
-    - CYCLONE_SERVER_HOST=http://172.18.0.1:7099
+    - CYCLONE_SERVER_HOST=http://172.18.0.6:7099
```
Comment (superxi911) [inline caicloud_e2e-test.yml]: revert

File: scripts/integration_image/Dockerfile [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
 
 COPY docker-entrypoint.sh /usr/local/bin/
 
-ENTRYPOINT ["docker-entrypoint.sh"]
-CMD ["sh"]
+EXPOSE 7099  
+EXPOSE 8000
 
+ENTRYPOINT ["docker-entrypoint.sh"]
+CMD ["sh"]
```
Comment (superxi911) [inline scripts/integration_image/Dockerfile]: /r/n

File: scripts/integration_image/docker-entrypoint.sh [hunk 4]
```diff
@@ -16,5 +17,4 @@ if [ -z "$DOCKER_HOST" -a "$DOCKER_PORT_2375_TCP" ]; then
 	export DOCKER_HOST='tcp://docker:2375'
 fi
 
-exec "$@"
-
+exec "$@"
```
Comment (superxi911) [inline scripts/integration_image/docker-entrypoint.sh]: /r/n

**Comment by xiaomuqiao** [comment]
LGTM

**Pull request merged**